### DISCUSSION TITLE
BurntToast v1 Image Functions: Combined or Separated?

### DISCUSSION THREAD

**Discussion by Windos** [Windos/BurntToast#192] [Ideas] [answered]
The rewrite of PowerShell in the leadup to v1.0.0 is well underway, and I need **your** input.

The current version of BurntToast has a single function for images, [New-BTImage](https://github.com/Windos/BurntToast/blob/main/BurntToast/Public/New-BTImage.ps1), which allows you to specify whether the image  is an app logo, hero, or regular image.

For the first public preview of BurntToast v1, there are now three functions that represent the three different types of images:

+ [Add-BTAppLogo](https://github.com/Windos/BurntToast/blob/v1.0.0-Preview1/src/Public/Add-BTAppLogo.ps1)
+ [Add-BTHeroImage](https://github.com/Windos/BurntToast/blob/v1.0.0-Preview1/src/Public/Add-BTHeroImage.ps1)
+ [Add-BTImage](https://github.com/Windos/BurntToast/blob/v1.0.0-Preview1/src/Public/Add-BTImage.ps1)

Which approach do you prefer? Which do you feel has a shallower learning curve? Am I completely off base and there's a third option I should consider?

**Comment by markekraus** [comment, marked as answer]
seems like these are simple and similar enough that one `Add-BTImage` would be more user-friendly and discoverable.

**Reply by Windos** [reply]
That's how the current version (pre-v1) works and I'll switch to doing that for v1 as well (was a good opportunity to test the alternative, but it turns out this time the right decision was the original one 😅)

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
